5.machine_learning/3.assess_generalizability/README.md

```diff
@@ -0,0 +1,57 @@
+# Assess generalizability of the models
```

Comment:
Quick question: I'm wondering why the UMAPs and precision recall curves are in the generalizability module? Shouldn't they be in the ML eval module? (happy to be wrong, but I wouldn't anticipate finding them here

Reply:
I have them included in here since these figures are related to the holdout plates which is apart of the "generalizability" of the models. The eval is just looking at the plate 4 data splits and looking at the performance for the data from that plate.

I will definitely change this structure in the next projects!